### PR TITLE
Update go.mod

### PR DESCRIPTION
**Pull request by urangel**
Update module path so that v2 resolves as a go module

### CODE REVIEW

**Comment by urangel** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by urangel** [comment]
We will also need to update the v2 tag with this change

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+    <td align="center"><a href="https://www.twitter.com/_wald0"><img src="https://avatars.githubusercontent.com/u/842644?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Andy Robbins</b></sub></a><br /><a href="#ideas-andyrobbins" title="Ideas, Planning, & Feedback">🤔</a> <a href="#design-andyrobbins" title="Design">🎨</a> <a href="#blog-andyrobbins" title="Blogposts">📝</a> <a href="#content-andyrobbins" title="Content">🖋</a> <a href="https://github.com/bloodhoundad/azurehound/v2/commits?author=andyrobbins" title="Documentation">📖</a></td>
+    <td align="center"><a href="https://github.com/ddlees"><img src="https://avatars.githubusercontent.com/u/8984872?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Dillon Lees</b></sub></a><br /><a href="https://github.com/bloodhoundad/azurehound/v2/commits?author=ddlees" title="Code">💻</a> <a href="#maintenance-ddlees" title="Maintenance">🚧</a> <a href="#ideas-ddlees" title="Ideas, Planning, & Feedback">🤔</a> <a href="#design-ddlees" title="Design">🎨</a></td>
```
Comment (ddlees) [inline CONTRIBUTORS.md]: Looks like you got a little over zealous with the `grep` `sed` replace magic 😉 

These links will result in a 404

File: README.md [hunk 2]
```diff
@@ -12,12 +12,12 @@ The BloodHound data collector for Microsoft Azure
 
 ### Release Binaries
 
-Download the appropriate binary for your platform from one of our [Releases](https://github.com/BloodHoundAD/AzureHound/releases).
+Download the appropriate binary for your platform from one of our [Releases](https://github.com/bloodhoundad/azurehound/v2/releases).
```
Comment (ddlees) [inline README.md]: Broken link

File: README.md [hunk 3]
```diff
@@ -12,12 +12,12 @@ The BloodHound data collector for Microsoft Azure
...
 
 #### Rolling Release
 
 The rolling release contains pre-built binaries that are automatically kept up-to-date with the `main` branch and can be downloaded from
-[here](https://github.com/BloodHoundAD/AzureHound/releases/tag/rolling).
+[here](https://github.com/bloodhoundad/azurehound/v2/releases/tag/rolling).
```
Comment (ddlees) [inline README.md]: Broken link

File: cmd/list-storage-containers.go [hunk 4]
```diff
@@ -67,7 +67,7 @@ func listStorageContainers(ctx context.Context, client client.AzureClient, stora
 		// Decreasing the demuxxer size only here is sufficient to prevent the cascade
 		// The error message with higher values for size is
 		// "The request was throttled."
-		// See issue #7: https://github.com/BloodHoundAD/AzureHound/issues/7
+		// See issue #7: https://github.com/bloodhoundad/azurehound/v2/issues/7
```
Comment (ddlees) [inline cmd/list-storage-containers.go]: Broken link

File: CONTRIBUTORS.md (hunk 1 above)
Comment (urangel) [reply CONTRIBUTORS.md]: Whoops, thank you for catching that! I have reinstated these

**Review by ddlees — APPROVED** [review]

**Pull request merged**